std/uni.d

```diff
@@ -8843,16 +8850,6 @@ void toLowerInPlace(C)(ref C[] s) @trusted pure
 {
     toCaseInPlace!(LowerTriple)(s);
 }
-// overloads for the most common cases to reduce compile time
-@safe pure /*TODO nothrow*/
```

Comment:
This really feel like anti-pattern/hack to me, and it's why I removed it. We have a template system in order to avoid things like this